readme.md

```diff
@@ -17,7 +17,7 @@ Asus-fan-control solves the problems above and even more. **Tested devices** wit
 | ASUS          | ASUS TUF Gaming | ASUS VivoBook       | ASUS ZenBook       | ASUSPRO      | ROG            |
 |---------------|-----------------|---------------------|--------------------|--------------|----------------|
 | [FX553VD][12] | [FX505GM][15]   | [15 X510UA][4]      | [Flip UX360UAK][6] | [B9440UA][1] | [G752VL][2]    |
-|               |                 | [15 X512FA][5]      | [UX410UA][7]       |              | [GL502VSK][10] |
+| [X507UF][17]  |                 | [15 X512FA][5]      | [UX410UA][7]       |              | [GL502VSK][10] |
```

Comment:
@dominiksalvet this is VivoBook too

Reply:
Yes, it's a Vivobook 15 series laptop. But, as Dominik mentioned in another issue, (https://www.asus.com/Laptops/For-Home/All-series/ASUS-X507), it's not Vivobook anymore.

Reply:
I will always respect official ASUS naming at the time of adding untested devices. And this case is no exception. Nevertheless, I must admit that even where I live some local distributors label it as a VivoBook (just as `dmidecode` reported). That will not change the presented name, though. Does this explanation make sense?